security/tpm/src/tpm.ts

```diff
@@ -3,22 +3,428 @@
 
 'use strict';
 import { errors } from 'azure-iot-common';
+import * as machina from 'machina';
+import * as tss from 'tss.js';
```

Comment:
don't need this if you import the types on the next line

Reply:
Aspect of what they are exporting.  Will bring up with msr after holidays.

---
In reply to: [157881300](https://github.com/Azure/azure-iot-sdk-node/pull/185#discussion_r157881300) [](ancestors = 157881300)